MapboxNavigation/RouteVoiceController.swift

```diff
@@ -203,8 +203,10 @@ open class RouteVoiceController: NSObject, AVSpeechSynthesizerDelegate, AVAudioP
     @objc open func didPassSpokenInstructionPoint(notification: NSNotification) {
         guard !NavigationSettings.shared.voiceMuted else { return }
         
-        let routeProgress = notification.userInfo![RouteControllerDidPassSpokenInstructionPointRouteProgressKey] as! RouteProgress
-        legProgress = routeProgress.currentLegProgress
+        routeProgress = notification.userInfo![RouteControllerDidPassSpokenInstructionPointRouteProgressKey] as? RouteProgress
```

Comment:
If `RouteControllerDidPassSpokenInstructionPointRouteProgressKey` exists and isn’t a RouteProgress, that’s an internal consistency error. Let’s assert that it’s a RouteProgress (or go back to using `!`).